pyproject.toml

```diff
@@ -24,8 +24,13 @@ DJANGO_SETTINGS_MODULE = "tin.settings"
 python_files = "tests.py test_*.py"
 norecursedirs = ["media", "migrations", "sandboxing"]
 testpaths = "tin"
-addopts = "--doctest-modules tin --import-mode=importlib -n 8"
+addopts = "--doctest-modules tin --import-mode=importlib -n auto"
 doctest_optionflags = "NORMALIZE_WHITESPACE NUMBER"
+filterwarnings = [
+    "error",
```

Comment:
Due to `pytest`'s bad naming, this is a bit confusing, but the `error` means to transform all warnings into exceptions. I'm not sure if this is wanted (or needed) but it should help during migrations between e.g. Django 4.2 - Django 5.2